### ISSUE TITLE
Only loads when page resolution is changes 

### ISSUE THREAD

**Issue by scottie** [AndreasFaust/react-ticker#55]
Something odd is happening , this component only loads when the page is made small from big or made big from small.
Thoughts ? I have tested this on 2 different applications and confirm both times this is the case.

**Comment by scottie** [comment]
No, it is a bug with the software package I had to move on it is clear this is no longer maintained.

**Comment by premtemp1** [comment]
@scottie 
did you use a different package?. I am having the same issue

**Comment by scottie** [comment]
Same issue here, developer is MIA lots of other packages on NPM to replace this broken one.
